### PR TITLE
Exchange Create Snapshot form

### PR DESCRIPTION
**Pull request by benrudolph**
A bit of a refactor for the snapshot form:
- Remove documentation file toggle
- Use crispy forms
- Add fieldsets based on this document: https://docs.google.com/document/d/1a9ZkT6FrMNQhKgUrUdHlvHdX7Ktnwtl3ds7diqoCEnc/edit#heading=h.bzwgrsvfhwkr

screencapture-localhost-8000-a-aspace-settings-project-snapshots-new-1429713062916
@millerdev

### CODE REVIEW

File: corehq/apps/domain/forms.py [hunk 1]
```diff
@@ -170,30 +177,57 @@ class SnapshotSettingsForm(forms.Form):
...
+
     video = CharField(label=ugettext_noop("Youtube Video"), required=False,
         help_text=ugettext_noop("An optional youtube clip to tell users about your app. Please copy and paste a URL to a youtube video"))
     documentation_file = forms.FileField(label=ugettext_noop("Documentation File"), required=False,
         help_text=ugettext_noop("An optional file to tell users more about your app."))
+    old_documentation_file = forms.BooleanField(required=False)
     cda_confirmed = BooleanField(required=False, label=ugettext_noop("Content Distribution Agreement"))
 
     def __init__(self, *args, **kw):
+        from corehq.apps.domain.views import CreateNewExchangeSnapshotView
```
Comment (dimagimon) [inline corehq/apps/domain/forms.py]: F401 'CreateNewExchangeSnapshotView' imported but unused

**Comment by millerdev** [comment]
Looks good to me, although not sure I have enough context to merge. Would it make sense to have the person who originally wrote the snapshot form take a look?

**Comment by benrudolph** [comment]
unfortunately i think the person who originally wrote it left the company (yedi), though i think @esoergel has some context on this project

**Comment by benrudolph** [comment]
(also fwiw, i wrote a test for this when i first submitted the change: https://github.com/dimagi/commcare-hq/blob/master/corehq/apps/domain/tests/test_views.py#L54, so it shouldn't be completely busted)

**Pull request merged**